.github/workflows/docpr.yml

```diff
@@ -0,0 +1,40 @@
+name: AutoDocumentPR
+
+on:
+  push:
+    branches: 
+      - dev
+      - master
+
+jobs:
+  dokka:
+    name: Dokka
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: "Cache ~/.gradle/caches"
+        uses: actions/cache@preview
+        with:
+          path: "~/.gradle/caches"
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/build.gradle.kts') }}
+          restore-keys: ${{ runner.os }}-gradle-
+
+      - name: Setup JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+          
+      - name: Run dokka
+        run: |
+          rm -rf dokka
+          ./gradlew dokka
+          
+      - name: Commit files
+        run: |
+          git add -f docs
+          git diff-index --quiet HEAD || git commit -m "Auto Document PR"
+     
+      - name: Push changes
+        run: git push
```

Comment:
maybe here should use `git push origin HEAD:<name-of-remote-branch>`
but Idk how to write the branch name here to map to the supposed one.

Reply:
I think it would be 
`git push origin HEAD:dev`